iati/validator.py

```diff
@@ -160,33 +184,73 @@ def extend(self, values):
             except TypeError:
                 pass
 
-    def contains_errors(self):
-        """Determine whether there are errors contained within the ErrorLog.
+    def get_errors(self):
```

Comment:
Aught to create an Issue to go through the library and make use of `get` prefixes consistent.

...somewhere like this it should almost be a read-only property than a function, though probably needs thinking about in more detail.

Reply:
Basically, `ValidationErrorLog`s need designing fully to provide a pleasant and comprehensive interface (which I very much didn't do upon their creation).